docs/guide/core-concepts.md

```diff
@@ -1,28 +1,188 @@
+Geographical Information Systems (GIS), like any specialized field, has a wealth
+of jargon and unique concepts. When represented in software, these concepts
+can sometimes be skewed or expanded from their original forms. Three of
+these critical to effective use of Geotrellis are *Projections*, *Extents*,
+and *Layout Definitions*. We give a thorough definition of many of the core concepts
```

Comment:
The sentence `Three of these critical to effective use of Geotrellis are *Projections*, *Extents*, and *Layout Definitions*.` can be removed.